packages/fxa-content-server/tests/functional/settings_v2/lib/helpers.js

```diff
@@ -24,7 +25,7 @@ async function navigateToSettingsV2(remote) {
 
   await openPage(EMAIL_FIRST, selectors.ENTER_EMAIL.HEADER, remote);
   await fillOutEmailFirstSignIn(email, password, remote);
-
+  await testElementExists(selectors.SETTINGS_V2.APP);
```

Comment:
The `remote` object needs to be passed as last param to these helper methods.

Reply:
how did this work for me locally?! 🤦